vm_test.go

```diff
@@ -149,9 +149,9 @@ func TestHyperRelocationNewcontainer(t *testing.T) {
 	err = vm.relocateHyperCommand(cmd)
 	assert.NotNil(t, err)
 
-	vm.Close()
```

Comment:
yes it is now a problem because we have the channel waiting for the end of the CTL channel in the CloseSockets(). That's why I had created a new function for this `WaitCtlEnd()`.